### PR TITLE
fix(e2e): use public.ecr.aws redis image

### PR DESCRIPTION
**Pull request by clamoriniere**
### What does this PR do?

Update test-infra dependency and end2end test since we will use another image for the redis test application.

### Motivation

avoid getting possibly image pull backoff during end2end test when the image was stored on dockerhub

### Additional Notes

related PR: https://github.com/DataDog/test-infra-definitions/pull/628

### Describe how to test/QA your changes

end2end test pass.

### CODE REVIEW

**Review by sblumenthal — APPROVED** [review]

File: test/new-e2e/tests/containers/ecs_test.go
Comment (jennchenn) [inline test/new-e2e/tests/containers/ecs_test.go:1]: would the `image_name` and `image_id`s  be unchanged? 
also here https://github.com/DataDog/datadog-agent/blob/d3c07fce4a0dc7eea2a79b9dd168c73c718f4cba/test/new-e2e/tests/containers/k8s_test.go#L364-L365

Comment (clamoriniere) [reply test/new-e2e/tests/containers/ecs_test.go:1]: yes you are right.

Comment (clamoriniere) [reply test/new-e2e/tests/containers/ecs_test.go:1]: PR updated

Comment (jennchenn) [reply test/new-e2e/tests/containers/ecs_test.go:1]: there are references to redis `image_name` in `k8s_test.go` as well https://github.com/DataDog/datadog-agent/blob/d3c07fce4a0dc7eea2a79b9dd168c73c718f4cba/test/new-e2e/tests/containers/k8s_test.go#L352-L383

Comment (clamoriniere) [reply test/new-e2e/tests/containers/ecs_test.go:1]: fixed it

**Review by pducolin — APPROVED** [review]

**Comment by pducolin** [comment]
Changelog: https://github.com/DataDog/test-infra-definitions/compare/6ba25fcf6a61...dfdd12299b97

**Review by L3n41c — APPROVED** [review]

**Pull request merged**